guides/common/modules/proc_adding-errata-to-an-incremental-content-view.adoc

```diff
@@ -4,10 +4,10 @@
 If errata are available but not installable, you can create an incremental Content View version to add the errata to your content hosts.
 For example, if the Content View is version 1.0, it becomes Content View version 1.1, and when you publish, it becomes Content View version 2.0.
 
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-errata-to-an-incremental-content-view[].
+IMPORTANT: If your content view version is old, you might encounter incompatibilities when incrementally adding enhanacement errata. 
+This is because enhancements are typically designed for the most current software in a repository.
```

Comment:
```suggestion
This is because enhancements are typically designed for the latest content view version in a repository.
```
If this is technically correct.

Reply:
@ianballou Can you please confirm that I can change this sentence as Malhar suggests?

Reply:
I do not believe so. Content View version is a different concept from Repositories.